search-commons/src/test/java/no/unit/nva/search/resource/ResourceClientTest.java

```diff
@@ -1153,7 +1152,53 @@ void shouldFindBySeriesIssn(
     assertThat(actualIdentifiers, containsInAnyOrder(expectedIdentifiers));
   }
 
-  private static @NotNull Set<String> extractActualIdentifiersFromHits(PagedSearch pagedResponse) {
+  @Test
```

Comment:
Tests doing actual tests against container using more or less the same ResourceSearchQuery builder settings. Not great, but the ResourceSearchQuery needs refactoring to be really testable.